hack/lib/build/binaries.sh

```diff
@@ -314,6 +315,41 @@ EOF
 }
 readonly -f os::build::generate_windows_versioninfo
 
+# Generates the .msi installer use for install oc.exe binary on windows
+function os::build::generate_windows_msi_installer() {
+  os::build::version::get_vars
+  local major="${OS_GIT_MAJOR}"
+  local minor="${OS_GIT_MINOR%+}"
+  local windows_wxs_file=`mktemp --suffix=".oc.wxs"`
+  cat <<EOF >"${windows_wxs_file}"
```

Comment:
I'd prefer to keep a template saved in this repo as `.xml` and edit it using `XMLPath` (or `sed`) -- that way, we get syntax highlighting for tools and can better inspect this file at rest.